components/ide/code/leeway.Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN sudo apt-get update \
     && sudo apt-get clean -y \
     && rm -rf /var/lib/apt/lists/*
 
-ENV GP_CODE_COMMIT ae8dc44299153ca820ac9e13cc9def5bb6baafd1
+ENV GP_CODE_COMMIT 334e0610dd1bb73487812b417b113d6a2b96e1b1
```

Comment:
**question**: Is this also adding the contextual menu on _Home Button_ click?

<img width="235" alt="Screenshot 2021-01-28 at 9 05 19 PM" src="https://user-images.githubusercontent.com/120486/106186493-066d6080-61ad-11eb-9a17-d90d6d48c205.png">

Reply:
no, it is behaviour of a new code, in next version there won't be `Home Button` anymore at all and contributed menu items will be merged in the main menu.